src/IW/Core/Url.hs

```diff
@@ -0,0 +1,8 @@
+module IW.Core.Url
+       ( Url (..)
+       ) where
+
+
+newtype Url = Url { unUrl :: Text }
```

Comment:
By style guide we do:

```haskell
newtype Url = Url 
    { unUrl :: Text 
    } deriving ...
```

Reply:
@vrom911 Yep, sorry completely overlooked that for some reason.